src/test/java/org/folio/roles/integration/kafka/CapabilityKafkaEventHandlerTest.java

```diff
@@ -52,6 +58,12 @@ class CapabilityKafkaEventHandlerTest {
   @Mock private FolioPermissionService folioPermissionService;
   @Mock private CapabilityEventProcessor capabilityEventProcessor;
   @Mock private CapabilitySetDescriptorService capabilitySetDescriptorService;
+  @Mock private PermissionOverrider permissionOverrider;
+
+  @BeforeEach
+  void setUp() {
+    lenient().when(permissionOverrider.getPermissionMappings()).thenReturn(null);
```

Comment:
Any test case when permission mappings are not empty?

Reply:
This was rewritten. Anyway the test case with non-empty mapping exists